circle.yml

```diff
@@ -1,12 +1,14 @@
 # This is the CircleCI.com configuration file to enable CI tests
-# It installs Lua, LuaRocks and LuaFileSystem on the CI worker, then downloads the Gallery plugin and the API descriptions
-# Finally it runs the Checker on the Gallery plugin
+# It installs Lua, LuaRocks and LuaFileSystem, lsqlite3, luasocket and luasec on the CI worker, then downloads the Gallery plugin and the AutoAPI descriptions
+# Finally it runs the Checker on the Gallery plugin, and on the internal MinimalPlugin test
```

Comment:
This looks like some merge conflict leftover, the MinimalPlugin is no longer present at all.